operators/src/machine_learning/onnx.rs

```diff
@@ -347,7 +374,62 @@ mod tests {
     };
     use ndarray::{Array1, Array2, arr2, array};
 
-    use super::*;
+    fn generate_model_metadata_from_onnx(
```

Comment:
shouldn't this method itself be tested to generate the correct metadata for the individual onnx files?

Reply:
[b1facd8](https://github.com/geo-engine/geoengine/pull/1000/commits/b1facd8ff3be816d9088e99465db1d043d4d9230)